src/objects.rs

```diff
@@ -1926,9 +1927,11 @@ mod tests {
         assert_eq!(Time::new(13, 37, 0), de("13:37:00").unwrap());
         assert_eq!(Time::new(0, 0, 0), de("0:0:0").unwrap());
         assert_eq!(Time::new(25, 42, 42), de("25:42:42").unwrap());
+        assert_eq!(Time::new(13, 37, 0), de("13:37:00.000").unwrap());
```

Comment:
Could you add a test on `"13:37:00.999"` just to be explicit (maybe tell it's not a requirement, but it's assumed that `currently we floor values`):
```suggestion
        assert_eq!(Time::new(13, 37, 0), de("13:37:00.000").unwrap());
        assert_eq!(Time::new(13, 37, 0), de("13:37:00.999").unwrap());  // currently floor on ms (not strictly required)
```